plugin-cgroupv2/src/k8s/utils.rs

```diff
@@ -33,23 +33,24 @@ pub struct CgroupV2MetricFile {
     pub node: String,
 }
 
-/// Check if a specific file is a dir. Used to know if cgroupv2 are used
+/// Check if a specific file is a dir. Used to know if cgroup v2 are used
 pub fn is_accessible_dir(path: &Path) -> bool {
     path.is_dir()
 }
 
 /// Returns a Vector of CgroupV2MetricFile associated to pods available under a given directory.
-fn list_metric_file_in_dir(
+fn kubernetes_list_metric_file_in_dir(
```

Comment:
Why did you prefixed this function with 'kubernetes' ? It is located in 'k8s' isn't it clear enough ?